### PR TITLE
Mitre indicators by open incidents bug fix

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/29292
fixes: https://github.com/demisto/etc/issues/29440

## Description
Changed the default time range in the dashboard to "lat 7 days" and add `to` and `from` time range to the query in MITREIndicatorsByOpenIncidents script

## Screenshots
Screen Shot 2020-10-18 at 14 39 49

Screen Shot 2020-10-18 at 14 39 57

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_4.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Dashboards
+##### MITRE ATT&CK
+- Updated the default time range to `last 7 days`.
+
+#### Scripts
+##### MITREIndicatorsByOpenIncidents
+- Added support for `to` and `from` time range arguments.
```
Comment (teizenman) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_4.md:8]: Add the docker image upgrade RN

File: Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py [hunk 2]
```diff
@@ -1,19 +1,43 @@
...
-return_outputs(temp)
+def main():
+    try:
+        from_date = demisto.args().get('from', '')
+        to_date = demisto.args().get('to', '')
+        query = get_query(from_date, to_date)
+        res = demisto.searchIndicators(query=query)
+
+        indicators = []
+        for ind in res['iocs']:
```
Comment (teizenman) [inline Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py:24]:
```suggestion
        for ind in res.get('iocs', []):
```

File: Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py [hunk 3]
```diff
@@ -1,19 +1,43 @@
...
+            indicators.append({
+                'Value': dict_safe_get(ind, ['value']),
+                'Name': dict_safe_get(ind, ['CustomFields', 'mitrename']),
+                'Phase Name': dict_safe_get(ind, ['CustomFields', 'mitrekillchainphases', 0, 'phase_name']),
+                'Description': dict_safe_get(ind, ['CustomFields', 'mitredescription']),
+
+            })
+
+        temp = tableToMarkdown('MITRE ATT&CK techniques by related Incidents', indicators,
```
Comment (teizenman) [inline Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py:33]: Change to a more meaningful name please

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_4.md (hunk 1 above)
Comment (barchen1) [reply Packs/FeedMitreAttack/ReleaseNotes/1_1_4.md]: done

File: Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py (hunk 2 above)
Comment (barchen1) [reply Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py]: done

File: Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py (hunk 3 above)
Comment (barchen1) [reply Packs/FeedMitreAttack/Scripts/MITREIndicatorsByOpenIncidents/MITREIndicatorsByOpenIncidents.py]: done

**Review by teizenman — APPROVED** [review]

**Pull request merged**